plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1882,6 +1885,7 @@ class account_failures {
    };
 
    std::map<account_name, account_failure> failed_accounts;
+   eosio::subjective_billing subjective_billing;
```

Comment:
`const eosio::subjective_billing&` we don't want a copy.

Reply:
Yeah I thought that as soon as I pushed the change.  Good catch. Thanks.